packages/components/select-v2/src/select-dropdown.vue

```diff
@@ -33,7 +34,7 @@ export default defineComponent({
     width: Number,
   },
   setup(props) {
-    const select = inject(selectV2InjectionKey)
+    const select = inject(selectV2InjectionKey) as SelectV2Context
```

Comment:
Why do you use `as SelectV2Context`?

Reply:
I may have made a mistake，there should be consistent with here as `any`
https://github.com/element-plus/element-plus/blob/a24c2977e15799601024d11b7988ecb9ddfc2e5a/packages/components/select-v2/src/select.vue#L278-L287